pages/kids/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { useTranslation } from 'next-i18next';
+import Head from 'next/head';
+import React, { useState } from 'react';
+import kidsWords from '../../data/translations/pro-deti.json';
+import { Button } from '../../components/basecomponents/Button';
+import { KidsTranslationsContainer } from '../../components/basecomponents/KidsTranslationContainer';
+export { getStaticProps } from '../../utils/localization';
+
+const Dictionary = () => {
```

Comment:
I would name this `Kids` or `KidsSection` we already use `Dictionary` for the main dictionary page.